src/main/java/lotto/view/ResultView.java

```diff
@@ -0,0 +1,43 @@
+package lotto.view;
+
+import lotto.domain.Lotto;
+
+import java.util.List;
+import java.util.Map;
+
+public class ResultView {
+
+    public static final String LOTTO_COUNT_OUTPUT = "%d개를 구매했습니다.";
+    public static final String GUIDELINE_FOR_WINNING = "\n당첨 통계\n--------";
+    public static final String WINNING_COUNT_OUTPUT = "%d개 일치 (%d원)- %d개";
+    public static final String PROFIT_RATE_OUTPUT = "총 수익률은 %f입니다.(기준이 1이기 때문에 결과적으로 손해라는 의미임)";
+
+    private ResultView() {
+        throw new AssertionError();
+    }
+
+    public static void printLottoCount(int lottoPrice) {
+        System.out.println(String.format(LOTTO_COUNT_OUTPUT, lottoPrice));
+    }
+
+    public static void printLottoNumber(List<Lotto> lottoList) {
+        lottoList.forEach(lotto -> System.out.println(lotto.getLotto()));
+    }
+
+    public static void printWinningCount(Map<Long, Long> winningCount, int lottoPrice) {
+        System.out.println(GUIDELINE_FOR_WINNING);
+        int[] winningMoney = {5000, 50000, 1500000, 2000000000};
+
+        float sum = 0;
+        for (long i = 0; i < winningMoney.length; i++) {
+            Long count = Long.valueOf(0);
```

Comment:
박싱된 기본타입을 사용할 필요가 있을까요?
```suggestion
            long count = 0;
```
이펙티브 자바에서는 `박싱된 기본 타입보다는 기본 타입을 사용하라` 라고 이야기 하고 있는데요 
한번 확인해보시면 좋을꺼 같아요! 😄 
https://jaehun2841.github.io/2019/03/01/effective-jave-item61/#%EC%9D%98%EB%8F%84%ED%95%98%EC%A7%80-%EC%95%8A%EC%9D%80-auto-boxing%EC%9C%BC%EB%A1%9C-%EC%9D%B8%ED%95%9C-%EC%84%B1%EB%8A%A5%EC%A0%80%ED%95%98

Reply:
오. 이런 글이 있었군요. 알려주셔서 감사합니다! 
그런데 코드를 보다 궁금한 점이 생겨서 질문 드립니다! long count의 경우 count = winningCount.get(i + 3); 코드에서 해시맵에서 value를 받아오면 Long이 되기 때문에 Auto Boxing이 될 것 같습니다. 어떤 코드가 더 효율적인지 궁금합니다!

Reply:
count 가 long 이면 언방싱이 되겠네요 😄 
박싱은 비싼 작업이지만 언박싱은 상대적으로 비용이 적습니다. 

한가지 count 가 long이면 winningCount 에 null 값이 있을 경우
Null 값에 대한 Exception이 발생할 수 있겠네요 😄